index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>
+            Twitter
+        </title>
+        <script src="https://kit.fontawesome.com/34321f5008.js" crossorigin="anonymous"></script>
+        <link href="https://fonts.googleapis.com/css2?family=Oxygen:wght@700&display=swap" rel="stylesheet">
+        <link rel="stylesheet" href="style.css" />
+        <script src="script.js" ></script>
```

Comment:
It's a best practice to put JS _script_ tags **before** the closing _/body_ tag rather than in the _head_ section

Reply:
At first, I did that but then I faced some problems so when I searched in google I found that you can solve this problem if you add the JS tag in the head section, and yeah it works !!